### PR TITLE
[multirobot - Part3] Add multi-robot launching

### PR DESCRIPTION
**Pull request by orduno**
---

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | # |
| Primary OS tested on | Ubuntu 18.04 |
| Robotic platform tested on | Gazebo simulation of TB3 |

---

## Description of contribution in a few bullet points

Extends #1147 by adding a launch file that:
* Starts a Gazebo world with no robots
* Spawns two robots into Gazebo
* Creates to namespaced instances of nav stack

### CODE REVIEW

File: nav2_bringup/bringup/params/nav2_params_namespaced.yaml [hunk 1]
```diff
@@ -0,0 +1,164 @@
+navigation_namespace:  # this is substituted with the actual robot name during launch
```
Comment (SteveMacenski) [inline nav2_bringup/bringup/params/nav2_params_namespaced.yaml]: I'm curious, can you load a .yaml into a namespace? That way you dont have to make identical yaml files that are only changed by the namespace the rest of the file is within?

Comment (orduno) [reply nav2_bringup/bringup/params/nav2_params_namespaced.yaml]: Will check if there's such feature in ROS.

Yaml doesn't support including another yaml :disappointed:

Comment (orduno) [reply nav2_bringup/bringup/params/nav2_params_namespaced.yaml]: @SteveMacenski I don't think this is currently supported by the [parser](https://github.com/ros2/rcl/blob/3e0efcef34b8da59bee0e5e0bcdb19577e6173cf/rcl_yaml_param_parser/include/rcl_yaml_param_parser/parser.h#L53)

Comment (orduno) [reply nav2_bringup/bringup/params/nav2_params_namespaced.yaml]: An alternative is to have a launch substitution which creates a namespaced YAML from the non-namespaced one. I'll try that.

Comment (orduno) [reply nav2_bringup/bringup/params/nav2_params_namespaced.yaml]: Pushed a commit that puts the yaml file on a namespace and got rid of the duplicated yaml file.

**Comment by orduno** [comment]
Any additional recommendations before merging?

File: nav2_bringup/bringup/params/nav2_params_namespaced.yaml (hunk 1 above)
Comment (SteveMacenski) [reply nav2_bringup/bringup/params/nav2_params_namespaced.yaml]: @orduno dude, go you, that's awesome. 

That's also something that really should be up stream but that's not important for this PR, just adding to my unending list of ecosystem tickets to fix

**Pull request merged**